homeassistant/components/modbus/__init__.py

```diff
@@ -131,6 +132,7 @@
         vol.Optional(
             CONF_SCAN_INTERVAL, default=DEFAULT_SCAN_INTERVAL
         ): cv.positive_int,
+        vol.Optional(CONF_LAZY_ERROR, default=0): cv.positive_int,
```

Comment:
Do we need a config option? Maybe just use a sane default?

Reply:
"need" is defined be the user 😄 In many situations especially when trying to locate an error (in the hardware etc), it is good to have "lazy_error: 0".

While in production, you most times do not really care if you miss 2-3 measurements, but having a nasty 0 in your graph is disturbing, in that case it is good to have "lazy_error: 3".

So I think a parameter is needed.

Reply:
I think there's two problem areas and both have better solutions. If there's a problem with the device reading I'd adjust the reading and if there's a problem with the how the graph draws the accumulated data points I'd adjust the data vizualization.

But ok.